lib/serialize.ml

```diff
@@ -133,7 +136,8 @@ module Writer = struct
   let schedule_chunk t iovecs =
     let length = IOVec.lengthv iovecs in
     write_chunk_length t.encoder length;
-    schedule_fixed t iovecs
+    schedule_fixed t iovecs;
+    write_crlf     t.encoder
```

Comment:
This is used to write the final empty chunk. With the change it will write to CRLFs to it, which isn't correct.

Reply:
According to the spec this is actually the correct behavior.

Reply:
Yeah, I thought it was as well.  It's consistent with the format of other chunks - one CRLF following the (0) size and one CRLF following the (empty) block of data.